docker/files/entrypoint.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import os
+from string import Template
+from subprocess import call
+
+from dokang.harvesters import sphinx_rtd_config
+from dokang_pdf import PdfHarvester
+
+VAR_UPLOAD_TOKEN = 'DOKANG_UPLOADED_DOCS_TOKEN'
+VAR_NAME = 'DOKANG_OPENSEARCH_NAME'
+VAR_DESCRIPTION = 'DOKANG_OPENSEARCH_DESCRIPTION'
+VAR_SERVER_TRUSTED_PROXY = 'DOKANG_SERVER_TRUSTED_PROXY'
+
+if not os.environ.get(VAR_UPLOAD_TOKEN):
```

Comment:
This test should be in `generate_dokang_config_file()` (it is irrelevant when we just use `sphinx_rtd_config_with_pdf()`)